docs/README.md

```diff
@@ -35,11 +37,72 @@ CI tests run against the following operating systems:
 - macOS Catalina 10.15,
 - Ubuntu 20.04 LTS.
 
-## Usage
+## Get Started
 
-See [DEVELOPING.md](DEVELOPING.md) for build and running instructions.
+### Installation
 
-See [USAGE.md](USAGE.md) for configuration instructions.
+You can download the binaries from
+[the artifacts of the `ci` GitHub workflow](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/actions/workflows/ci.yml?query=branch%3Amain).
+
+In the documentation,
+the binaries location is refered as `%InstallationLocation%`.
+
+On Linux, after the installation,
+you can optionally create the default log directory:
+
+```sh
+sudo mkdir -p /var/log/opentelemetry/dotnet
+sudo chmod a+rwx /var/log/opentelemetry/dotnet
+```
+
+### Instrument a .NET application on Windows
+
+Before running the application, set the following environment variables:
+
+```env
+COR_ENABLE_PROFILING=1
+COR_PROFILER={918728DD-259F-4A6A-AC2B-B85E1B658318}
+COR_PROFILER_PATH_64=%InstallationLocation%/win-x64/OpenTelemetry.AutoInstrumentation.Native.dll
+COR_PROFILER_PATH_32=%InstallationLocation%/win-x86/OpenTelemetry.AutoInstrumentation.Native.dll
+CORECLR_ENABLE_PROFILING=1
+CORECLR_PROFILER={918728DD-259F-4A6A-AC2B-B85E1B658318}
+CORECLR_PROFILER_PATH_64=%InstallationLocation%/win-x64/OpenTelemetry.AutoInstrumentation.Native.dll
+CORECLR_PROFILER_PATH_32=%InstallationLocation%/win-x86/OpenTelemetry.AutoInstrumentation.Native.dll
```

Comment:
We should probably add `DOTNET_STARTUP_HOOKS=%InstallationLocation/netcoreapp3.1/OpenTelemetry.AutoInstrumentation.StartupHook.dll` to this list, because it's required for bootstrapping the SDK for .net core apps.

And we might also want to add the additional deps variable too.
`DOTNET_ADDITIONAL_DEPS=%InstallationLocation/AdditionalDeps`

Reply:
Would it also be good to add it to `dev/envvars.sh`?

Reply:
The startup hook is already in there. https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/0d0da3562a4efae89973238ecdc98f2fdda36013/dev/envvars.sh#L56
But the additional deps is not, and maybe it should be.

Reply:
Addressed d34ca1b4